config/routes.rb

```diff
@@ -1,13 +1,17 @@
 Rails.application.routes.draw do
+
   devise_for :users
   
   get 'welcome/index'
   get 'welcome/about'
-
   root to: 'welcome#index'
   
   post :incoming, to: 'incoming#create'
 
-  resources :topics
   resources :users, only: [:update, :show, :index]
+  
+  resources :topics do
+    resources :bookmarks, except: [:index]
```

Comment:
Confused why index was removed here. The way you have it coded, you have an index controller method and an index view, @JakeWengroff.